src/utils/storage.spec.ts

```diff
@@ -0,0 +1,27 @@
+import Nodee from '../list/node';
```

Comment:
don't use Nodee for testing, use a plain object, in order to decouple storage implementation from nodee.